src/pages/en/guides/content-collections.mdx

```diff
@@ -396,6 +396,127 @@ const { Content } = await entry.render();
 <Content />
 ```
 
+## Migrating from File-Based Routing 
+If your existing site uses [file-based routing](/en/core-concepts/astro-pages/#markdownmdx-pages) with Markdown files in the `src/pages` folder, you may want to migrate to content collections. This gives you the benefit of type-safe frontmatter, but requires you to set up routing yourself. This guide uses the result of the [Build a Blog tutorial](/en/tutorial/0-introduction/) as an example starting point.
```

Comment:
If we leave this guide here, I'm onboard with this change, as we discussed the benefits of content collections already. But if we want to move this into a self-contained recipe, providing some motivation seems worthwhile.

Reply:
Agreed! For here, though, I say let's remove this info, (in part because I want to avoid thinking about exactly how to position this and what kind of caveat to include in this introduction) and we'll work it back in when it goes somewhere else!  :raised_hands: